packages/esm-appointments-app/src/appointment-workload/workload-component.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Dropdown, DataTableSkeleton } from '@carbon/react';
+
+import styles from './workload-card.scss';
+import WorkloadCard from './workload-card-component';
+
+function Workload() {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className={styles.cardContainer}>
+        <WorkloadCard label={'100'} headerLabel={'12/6'} />
```

Comment:
Does this mock data have backend support yet? Some static data is ok to get the component working, but I would prefer to see the data being passed through a data-fetcher which is mocking the response payload.